From c41afc4210021b07458361c38580979cca53c4a6 Mon Sep 17 00:00:00 2001
From: Zhao Wei Liew <zhaoweiliew@gmail.com>
Date: Sun, 2 Oct 2016 07:33:26 +0800
Subject: [PATCH 13/19] Settings: battery: Add LineageParts perf profiles

Replace the original battery saver preference with the performance
profiles preference in LineageParts. We'll encapsulate all settings
related to battery profiles within the preference.

Change-Id: I240e7f84838677b3755fc28cee4e982700829271
---
 res/xml/power_usage_summary.xml | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/res/xml/power_usage_summary.xml b/res/xml/power_usage_summary.xml
index d8003dfd1d..210effdc3a 100644
--- a/res/xml/power_usage_summary.xml
+++ b/res/xml/power_usage_summary.xml
@@ -16,6 +16,7 @@
 
 <PreferenceScreen
     xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:lineage="http://schemas.android.com/apk/res/lineageos.platform"
     xmlns:settings="http://schemas.android.com/apk/res-auto"
     android:key="power_usage_summary_screen"
     android:title="@string/power_usage_summary_title"
@@ -44,6 +45,11 @@
         android:title="@string/smart_battery_manager_title"
         settings:controller="com.android.settings.fuelgauge.batterytip.BatteryManagerPreferenceController" />
 
+    <!-- Custom perf profiles settings -->
+    <org.lineageos.internal.lineageparts.LineagePartsPreference
+        android:key="perf_profile_settings"
+        lineage:replacesKey="battery_saver_summary" />
+
     <SwitchPreference
         android:key="battery_percentage"
         android:title="@string/battery_percentage"
-- 
2.17.1

